src/v2/Apps/Conversation/Utils/usePoll.ts

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useRef } from "react"
+
+type IntervalFunction = () => unknown | void
+
+export default function usePoll(
+  callback: IntervalFunction,
+  seconds: number,
+  key: string | null,
+  stop?: boolean
+) {
```

Comment:
Totally non-blocking form suggestion: We should use a single object as the argument (like how `props` works in a react component), versus a series of unnamed "bare" args. 

For example, a future dev at a glance will have a hard time understanding what each slot does ("what does '10' mean 🤔 ):  

```tsx
usePoll(
    () => {
      relay.refetch({ conversationID: conversation?.internalID }, null)
    },
    10,
    match.params?.conversationID,
    !tabActive
  )
```

Where something like the below, while more verbose, leaves no room for ambiguity:

```tsx
usePoll({
    callback: () => {
      relay.refetch({ conversationID: conversation?.internalID }, null)
    },
    intervalTime: 10,
    clearWhen: !tabActive 
    key: match.params?.conversationID,
  })
```

By using objects as args the code becomes self-documenting (since an object's keys describe its function), closer to React's idioms (which we spend the most time in) as well as less brittle (since you can move object keys all around, where naked args have fixed positions).

Reply:
Yeah, makes sense 👍